production/ksonnet/README.md

```diff
@@ -17,6 +17,10 @@ In your config repo, if you don't yet have the directory structure of Tanka set
 $ mkdir config && cd config/
 $ tk init
 $ tk env add loki --namespace=loki
+$ tk env set loki --server=https://$k8s_master_node_ip:6443
```

Comment:
`s#loki#environments/loki#`

Reply:
If so, must be `tk env add environment/loki`

Reply:
Yes! This is correct, I introduced a mistake here